doc/sphinx/changes.rst

```diff
@@ -6,6 +6,470 @@ Recent changes
 
    .. include:: ../unreleased.rst
 
+Version 8.11
+------------
+
+Summary of changes
+~~~~~~~~~~~~~~~~~~
+
+The main changes brought by |Coq| version 8.11 are:
+
+- `Ltac2`__, a new tactic language for writing more robust larger scale
+  tactics, with built-in support for datatypes and the multi-goal tactic monad.
+- `Primitive floats`__ are integrated in terms and follow the binary64 format
+  of the IEEE 754 standard, as specified in the `Coq.Float.Floats` library.
+- `Cleanups`__ of the section mechanism, delayed proofs and further
+  restrictions of template polymorphism to fix soundness issues related to
+  universes.
+- New `unsafe flags`__ to disable locally guard, positivity and universe
+  checking. Reliance on these flags is always printed by
+  :g:`Print Assumptions`.
+- `Fixed bugs`__ of :g:`Export` and :g:`Import` that can have an impact
+  on user developments.
+- New interactive development method based on `vos` `interface files`__,
+  allowing to work on a file without recompiling the proof parts of their
+  dependencies.
+- New :g:`Arguments` annotation for `bidirectional type inference`__
+  configuration for reference (e.g. constants, inductive) applications.
+- Generalization of the :g:`under` and :g:`over` tactics__ of SSReflect to
+  arbitrary relations.
+- `Revision`__ of the :g:`Coq.Reals` library, its axiomatisation and
+  instances of the constructive and classical real numbers.
+
+__ 811Ltac2_
+__ 811PrimitiveFloats_
+__ 811Sections_
+__ 811UnsafeFlags_
+__ 811ExportBug_
+__ 811vos_
+__ 811BidirArguments_
+__ 811SSRUnderOver_
+__ 811Reals_
+
+The ``dev/doc/critical-bugs`` file documents the known critical bugs of |Coq|
+and affected releases. See the `Changes in 8.11+beta1`_ section for the
+detailed list of changes, including potentially breaking changes marked with
+**Changed.**.
```

Comment:
The repetition of `.` is weird. Moreover, Gaëtan has suggested making that a `:`, which I've already implemented in #10931.

```suggestion
**Changed**.
```